java/test/jmri/jmrit/display/layoutEditor/LayoutEditorDialogs/LayoutTrackEditorsTest.java

```diff
@@ -801,11 +795,11 @@ public void testEditTurntableErrors() {
     }
 
     public void createTurnouts() {
-        turnout0 = InstanceManager.getDefault(TurnoutManager.class).provideTurnout("IT101");
+        turnout0 = InstanceManager.getDefault(jmri.TurnoutManager.class).provideTurnout("IT101");
```

Comment:
Once you've added the `import jmri.*`, the `jmri.` prefix isn't needed here.  Before the `import jmri.*` was added, `jmri.TurnoutManager` wasn't being imported.

Reply:
I was getting an error (on Travis CI) that it couldn’t find the TurnoutManager.class… I added the jmri prefix specifically to fix that.

Reply:
I’ll check the import… Thanks.